templates/release-semantic-automated-npm/release.config.ejs

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  "branches": <%- branches %>,
+  "plugins": [
+    "@semantic-release/commit-analyzer",
+    "@semantic-release/release-notes-generator",
+    "@semantic-release/changelog",
+    ["@semantic-release/npm", {
+      "tarballDir": "release",
+      "npmPublish": false
+    }],
+    <% if (mainFilename) { %>
+     ["@semantic-release/exec", {
+        "prepareCmd": "sed -i \"s/Version:.*/Version:     ${nextRelease.version}/\" \"<%- mainFilename %>\" ",
+      }],
+    <% } %>
+    "@semantic-release/github",
+    ["@semantic-release/git", {
+      "assets": <%- filesToCommit %>,
```

Comment:
Why not hardcode this? `@semantic-release/git` should be smart enough to ignore non-existing files.
```suggestion
      "assets": ["CHANGELOG.md", "composer.json", "index.php", "package-lock.json", "package.json", "style.css"],
```

Reply:
Done